### ISSUE TITLE
Refactor Email Send Logic

### ISSUE THREAD

**Issue by a6wu** [UCSD/campus-mobile#138]
Move email send logic from mobile app to a cloud service.

**Comment by i8wu** [comment]
You mean the feedback form submit?

**Comment by a6wu** [comment]
Yes

**Comment by i8wu** [comment]
Does this actually need to be moved? Would be posting to AWS which then posts to eforms.

**Comment by c3bryant** [comment]
Since MachForms handles the actual sending of the email, this may not be required, let's revisit

**Comment by a6wu** [comment]
Can the send feature in the app itself be async so user aren't blocked on the UI while the network request is made?

**Comment by c3bryant** [comment]
Closing this issue, created a new issue #157 to look into a non-blocking UI
